Sources_v3/StreamChatUI/ChatChannel/MessageComposerSuggestionsViewController.swift

```diff
@@ -17,6 +51,12 @@ open class MessageComposerSuggestionsViewController<ExtraData: UIExtraDataTypes>
         }
     }
 
+    public var suggestionsCommands: [MessageComposerSuggestionCommand] = [.giphy, .imgur, .customCommand] {
+        didSet {
+            updateContent()
```

Comment:
`updateContentIfNeeded()` -> https://www.notion.so/UIConfig-and-View-Structure-bdbd703e4ba44e1c8039b43ccf17f56f#38a2c96b14794e449a51c0370dfbffae

Reply:
Deleted already